system/bind/templates/etc/_secret_env.tpl

```diff
@@ -0,0 +1,7 @@
+{{- if .Values.secret_env }}
+{{- range .Values.secret_env }}
```

Comment:
I think there is an extra `env` key in the dictionary.

Reply:
Not under the ```secret-env:``` key.